### PR TITLE
BS-178 | Increased default tomcat timeout

### PR DESCRIPTION
**Pull request by daviemukungi**
## Requirements

- [x] This PR has a proper title that briefly describes the work done
- [x] I have squashed / amended the comments to make it more redable
- [x] I have included link to all the JIRA ticket('s)
- [x] I wrote the code as a pair or atleast performed a self-review of my own code
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

## Summary
Prolonged the default Tomcat timeout to 10 minutes to address the needs of SNOMED reports that involve extended response times specifically the SNOMED Clinical Finding diagnosis(ICD) line report.

## Screenshots
clinicalFindings

## JIRA tickets
https://bahmni.atlassian.net/browse/BS-178

### CODE REVIEW

File: src/main/java/org/bahmni/reports/template/TSIntegrationDiagnosisService.java [hunk 1]
```diff
@@ -19,6 +19,7 @@
 import java.sql.Statement;
 import java.text.MessageFormat;
 import java.util.ArrayList;
+import java.util.Date;
```
Comment (sivareddyp) [inline src/main/java/org/bahmni/reports/template/TSIntegrationDiagnosisService.java:22]: remove this unused import

Comment (daviemukungi) [reply src/main/java/org/bahmni/reports/template/TSIntegrationDiagnosisService.java]: @sivareddyp this has been resolved

File: package/docker/bahmni-reports/template/bahmni-reports.properties.template [hunk 2]
```diff
@@ -10,8 +10,8 @@ reports.password=${REPORTS_DB_PASSWORD}
 openmrs.service.rootUrl=http://${OPENMRS_HOST}:${OPENMRS_PORT}/openmrs/ws/rest/v1
 openmrs.service.user=reports-user
 openmrs.service.password=Admin123
-openmrs.connectionTimeoutInMilliseconds=30000
-openmrs.replyTimeoutInMilliseconds=120000
+openmrs.connectionTimeoutInMilliseconds=600000
```
Comment (sivareddyp) [inline package/docker/bahmni-reports/template/bahmni-reports.properties.template:13]: @daviemukungi this property need not to be changed. Only read timeout need to be updated.

Comment (daviemukungi) [reply package/docker/bahmni-reports/template/bahmni-reports.properties.template]: This has been resolved @sivareddyp

**Review by sivareddyp — APPROVED** [review]

**Review by vijayanandtwks — APPROVED** [review]

**Pull request merged**